server/src/updaters/relationship-updaters.js

```diff
@@ -40,7 +41,11 @@ async function updateRelationships(
     }
   }
 
-  if (action === 'send_friend_request') {
+  if (!userIDs.length) {
+    return errors;
+  }
+
+  if (relationshipActions.FRIEND) {
```

Comment:
You forgot the `action ===` here. Would be awesome if we could catch small errors like this before review. Making sure you test the endpoint will help a lot, as will *carefully* going through every line in your PR before putting it up.